orca-kayenta/src/main/kotlin/com/netflix/spinnaker/orca/kayenta/pipeline/CleanupCanaryClustersStage.kt

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.kayenta.pipeline
+
+import com.netflix.spinnaker.orca.clouddriver.pipeline.cluster.DisableClusterStage
+import com.netflix.spinnaker.orca.clouddriver.pipeline.cluster.ShrinkClusterStage
+import com.netflix.spinnaker.orca.kayenta.model.Deployments
+import com.netflix.spinnaker.orca.kayenta.model.deployments
+import com.netflix.spinnaker.orca.kayenta.model.regions
+import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder
+import com.netflix.spinnaker.orca.pipeline.WaitStage
+import com.netflix.spinnaker.orca.pipeline.graph.StageGraphBuilder
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import org.springframework.stereotype.Component
+
+@Component
+class CleanupCanaryClustersStage : StageDefinitionBuilder {
+  companion object {
+    @JvmStatic
+    val STAGE_TYPE = "cleanupCanaryClusters"
+  }
+
+  override fun beforeStages(parent: Stage, graph: StageGraphBuilder) {
+    val deployments = parent.deployments
+
+    // disable control cluster
+    val disableStage1 = graph.add {
```

Comment:
Why not just name these vals what they are, rather than pairing with a comment? `disableControlStage`, etc.